BingMaps/rest-services/directly-accessing-the-bing-maps-tiles.md

```diff
@@ -1,5 +1,6 @@
 ---
-title: "Directly accessing the Bing Maps tiles | Microsoft Docs"
+title: "Directly Accessing the Bing Maps Tiles | Microsoft Docs"
+description: "This article outlines direct access the Bing Map tiles and how to add Bing Maps as a tile layer in a non-Bing Maps control."
```

Comment:
This article outlines direct access to the Bing Map tiles and how to add Bing Maps as a tile layer in a non-Bing Map control.